### ISSUE TITLE
Spam DNS responses

### ISSUE THREAD

**Issue by p-b--** [TheRook/subbrute#12]
It looks like the following two IP address generate spam responses that evade your detection mechanism.
```
207.69.188.189
207.69.188.184
```

Some cursory testing showed me the following:
Using your method:
```
$ nslookup 0f3698d0d478470a9ccbc2960324a21a.com. 207.69.188.184
*** rns0.earthlink.net can't find 0f3698d0d478470a9ccbc2960324a21a.com: Non-existent domain
Server:  rns0.earthlink.net
Address:  207.69.188.184
```
Using just a long domain name:
```
$ nslookup superlongfakedomainthatexceedsthelengthoftheuuid.com. 207.69.188.184
Non-authoritative answer:
Server:  rns0.earthlink.net
Address:  207.69.188.184

Name:    superlongfakedomainthatexceedsthelengthoftheuuid.com
Address:  92.242.140.1

```
Fake subdomain:
```
$ nslookup fake.example.com. 207.69.188.184
Non-authoritative answer:
Server:  rns0.earthlink.net
Address:  207.69.188.184

Name:    fake.example.com
Address:  92.242.140.1
```

It appears that they are somehow detecting that the uuid method and not returning anything for that, but returning the spam result for anything else.

**Comment by bottlsecteam** [comment]
Hey TheRook, 
Thanks for sharing that. I whipped this up a while back and haven't re-visited it's functions so I'm sure there's caveats.

**Comment by TheRook** [comment]
Good find.  There are some spammy DNS servers that seem to be non-deterministic in their polluted results.   I have also gotten a report of a shady ISP's that is intercepting failed DNS responses and responding with spammy 'A' records.  My ISP doesn't do this and I haven't been able to test this error condition first hand.

Commit 57 removes spammy resolvers found on 207.69.188.*.  This should make results more accurate,  but the underlying problem is still unsolved so I'm going to leave this bug open.  Clearly more work needs to be done on SubBrute's wildcard filter.

**Comment by TheRook** [comment]
Solved in SubBrute 2.0 by looking at return code, as well as preferring authoritative over open resolvers. 

If this issue wasn't solved,  let me know!
